keps/sig-storage/1495-volume-populators/README.md

```diff
@@ -271,6 +294,153 @@ downgrade.
+  existence of any VolumePopulator.populator.storage.k8s.io CRs would indicate
+  that a populator is installed and could be used.
+
+* **What are the SLIs (Service Level Indicators) an operator can use to determine 
+the health of the service?**
+  - [ ] Metrics
+    - Metric name:
+    - [Optional] Aggregation method:
+    - Components exposing the metric:
+  - [ ] Other (treat as last resort)
+    - Details:
+
+* **What are the reasonable SLOs (Service Level Objectives) for the above SLIs?**
+  At a high level, this usually will be in the form of "high percentile of SLI
+  per day <= X". It's impossible to provide comprehensive guidance, but at the very
+  high level (needs more precise definitions) those may be things like:
+  - per-day percentage of API calls finishing with 5XX errors <= 1%
+  - 99% percentile over day of absolute value from (job creation time minus expected
+    job creation time) for cron job <= 10%
+  - 99,9% of /health requests per day finish with 200 code
+
+* **Are there any missing metrics that would be useful to have to improve observability 
+of this feature?**
+  Describe the metrics themselves and the reasons why they weren't added (e.g., cost,
+  implementation difficulties, etc.).
+
+### Dependencies
+
+* **Does this feature depend on any specific services running in the cluster?**
+  This feature depends on the VolumePopulator CRD being installed, and the
+  associated validating webhook for PVCs.
+
+### Scalability
+
+* **Will enabling / using this feature result in any new API calls?**
+  No
+
+* **Will enabling / using this feature result in introducing new API types?**
+  Describe them, providing:
+  - API type: VolumePopulator.populator.storage.k8s.io
+  - Supported number of objects per cluster: Small number
+  - Supported number of objects per namespace: Not namespaced
+
+* **Will enabling / using this feature result in any new calls to the cloud 
+provider?**
+  No
+  
+* **Will enabling / using this feature result in increasing size or count of 
+the existing API objects?**
+  No
+
+* **Will enabling / using this feature result in increasing time taken by any 
+operations covered by [existing SLIs/SLOs]?**
+  Creation of PVCs will have to go through an additional validation webhook.
+
+* **Will enabling / using this feature result in non-negligible increase of 
+resource usage (CPU, RAM, disk, IO, ...) in any components?**
+  The new validation webhook will consist of a deployment with 1 or more
+  replicas.
+
+### Troubleshooting
+
+* **How does this feature react if the API server and/or etcd is unavailable?**
+  n/a
+  
+* **What are other known failure modes?**
+  For each of them, fill in the following information by copying the below template:
+  - [Failure mode brief description]
+    - Detection: How can it be detected via metrics? Stated another way:
+      how can an operator troubleshoot without logging into a master or worker node?
```

Comment:
Metrics on volume provisioning
Events on PVCs

Reply:
> Metrics on volume provisioning
> Events on PVCs

Also, metrics on your validating admission webhook (up or down), right?

Reply:
I don't see a list of metrics here.